src/assets/stylesheets/main/_sphinx.scss

```diff
@@ -88,3 +88,12 @@
     font-style: italic;
   }
 }
```

Comment:
This should be moved up so that it is within the last curly bracket. Thus, the compiled CSS selectors would be more specific to HTML generated by this theme: `.md-typeset div.line-block`

Reply:
You might also get bit by our SASS style checker. Just make sure there's a blank line before the new rules (contained with the last curly bracket) and use 2-space indents.